docs/astro/sql/operators/load_file.rst

```diff
@@ -0,0 +1,162 @@
+======================================
+load_file operator
+======================================
+
+When to Use load_file operator
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
It is probably worth referencing/include here what we support:
- databases
- file locations
- file types

Reply:
Done.